src/app/components/form/input/FormInputPassword.vue

```diff
@@ -12,33 +12,57 @@
       <Progress :model-value="strength" class="my-2" />
     </template>
     <template #icon>
-      <IHeroiconsEye v-if="!isVisible" />
-      <IHeroiconsEyeSlash v-else />
+      <IVibetypeClose
+        v-if="
+          (formInput.sameAs && formInput.sameAs.$invalid && formInput.$dirty) ||
+          (formInput.lengthMin &&
+            formInput.lengthMin.$invalid &&
+            formInput.$dirty)
+        "
+        :aria-label="t('iconAltClose')"
+        class="h-6 w-6 text-(--semantic-critic-text)"
+        :title="t('validNot')"
```

Comment:
```suggestion
        class="size-6 text-(--semantic-critic-text)"
```

I think `aria-label` is enough, or do we need the title too?

Reply:
I think its best to have both, title will act as a tool tip, aria-label is for screen readers.

Reply:
![image](https://github.com/user-attachments/assets/13aaa113-b66a-479a-b914-f4d45160929c)

I don't get a tooltip though, do you? :thinking:

Reply:
![Screenshot From 2025-04-14 21-39-51](https://github.com/user-attachments/assets/9ce6bbef-390d-469a-8105-f2ee05d15b40)

I don't get a tooltip though, do you? :thinking:

Reply:
We agreed that the title is not necessary.